app/bundles/PluginBundle/Config/config.php

```diff
@@ -118,6 +118,10 @@
                 'class'     => 'Mautic\PluginBundle\Helper\IntegrationHelper',
                 'arguments' => 'mautic.factory',
             ],
+            'mautic.helper.plugin.builder' => [
+                'class'     => 'Mautic\PluginBundle\Helper\BuilderHelper',
+                'arguments' => 'mautic.factory',
```

Comment:
I think that Alan meant especially this dependency. Let's not create new classes with the MauticFactory dependencies.